frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -975,36 +981,41 @@ class Worksheet extends React.Component {
                     ) {
                         // If the number of bundles increases then the focus should be on the new bundle.
                         // if the current focus is not on a table
-                        if (items[this.state.focusIndex] &&
-                            items[this.state.focusIndex].mode &&
-                            items[this.state.focusIndex].mode !== 'table_block') {
-                            this.setFocus(this.state.focusIndex + 1, 0);
+                        if (items[focus] &&
+                            items[focus].mode &&
+                            items[focus].mode !== 'table_block') {
+                            this.setFocus(focus >= 0 ? focus + 1 : 'end', 0);
```

Comment:
Why add this conditional? Wouldn't passing in -1 to the first argument of `setFocus` have the same affect as passing in "end"?

Reply:
Nope. One would deselect the element whereas passing in "end" would select the last element.